src/vsr/replica.zig

```diff
@@ -5496,37 +5578,29 @@ pub fn ReplicaType(
 
             var headers_canonical = DVCQuorum.headers_canonical(self.do_view_change_from_all_replicas);
             const header_head = headers_canonical.next().?;
-            assert(header_head.op >= do_view_change_commit_min_max);
```

Comment:
Replaced by `for (dvcs_all.constSlice()) |dvc| assert(header_head.op >= dvc.header.commit);` which is the same check, just inline.